src/trans/gpu/internal/ltinv_mod.F90

```diff
@@ -386,6 +386,7 @@ SUBROUTINE LTINV(ALLOCATOR,HLTINV,KF_UV,KF_SCALARS,&
     ENDIF
 
 #ifdef OMPGPU
+    !$OMP TASKWAIT
```

Comment:
Shouldn't this be paired with a `NOWAIT`?

Reply:
You're right, I guess I must have thrown this in due to the presence of `ACC WAIT(1)` in the corresponding directive below, but your comment also applies to the OpenACC branch, as I do not see any `ASYNC` statements here. I'm disabling this but leaving a comment for later.